forestserviceprototype/static_assets/additional.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function(){
+  $('.add_another_permit_holder').click(function(e){
```

Comment:
moving the hiding of any fields into `js` instead of css, so that without js enabled all the fields should be enabled.